misc/config_tools/upgrade_scenario_xml

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+function usage() {
+	echo "Usage: $0 <scenario XML> [<version of the given scenario XML>]"
+	echo "Upgrade the given scenario XML to align with the current version of ACRN."
+	echo ""
+	echo "Example: $0 logical_partition.xml v2.3"
+	echo "    The output file is logical_partition-v2.3.xml"
+}
+
+file=$1
+ver=$2
+
+if [[ "${file}" = "" ]]; then
+	usage
+	exit 1
+fi
+
+if [[ ! -f "${file}" ]]; then
+	echo "${file}: No such file"
+	exit 1
+fi
+
+if [[ ! "${ver}" =~ v[0-9]+.[0-9]+ ]]; then
+	echo "${ver}: Invalid version number"
+	echo "Example of valid version number: v2.0"
```

Comment:
I think it would be better to indicate a supported version here, i.e. v2.3 or v2.4. I'd go with `v2.3` since this is also what's used in the `usage()` function.
```suggestion
	echo "Example of valid version number: v2.3"
```

Reply:
Will update in v2.